ImperatorToCK3.UnitTests/PopTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using commonItems;
+using Xunit;
+
+namespace ImperatorToCK3.UnitTests {
+    public class PopTests {
+        [Fact] public void IDCanBeSet() {
+            var reader = new BufferedReader(" = {}");
+            var thePop = new Imperator.Pops.PopFactory().GetPop("42", reader);
+            Assert.Equal((ulong)42, thePop.ID);
+        }
+        [Fact]
+        public void CultureCanBeSet() {
```

Comment:
I've developed a preference for Zemurin's habit of combining tests into ones that verify everything is as default, and ones that verify everything can be set.